### PR TITLE
more on Flutter web indexability

### PR DESCRIPTION
**Pull request by yjbanov**
Expand on what we're doing about indexability.

One unusual aspect of this change is linking to the test site and spelling out "Hawaii-themed space stories". This is intentional, and hopefully, not too disruptive to the message. In order to prove that SEO works we need an already indexed website to link to the test site so search engines find it and attempt to index it. I thought sneaking one such link into a section of our web FAQ that already talks about SEO would be a safe place for it.

One place I violate the style guide is I use "we" in the text. However, it seems to be consistent with how the word is used elsewhere on the site. I'm open to suggestions for how to remove it, if it's important.

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by kevmoo — APPROVED** [review]

**Review by johnpryan — APPROVED** [review]

File: src/content/platform-integration/web/faq.md
Comment (atsansone) [inline src/content/platform-integration/web/faq.md:41]: **suggestion:** Would recommend tightening somewhat as the quantity of "we"s is a bit much.

```suggestion
That said, as mentioned in the [roadmap][], the Flutter team plans to investigate
search engine indexability of Flutter Web.
To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii].
We intend for search engines to find and index this site.
```

**Review by atsansone — APPROVED** [review]
Nothing wrong with the occasion "we", but I suggested a change to not add as many.

Acceptance is your call.

File: src/content/platform-integration/web/faq.md
Comment (sfshaza2) [reply src/content/platform-integration/web/faq.md:41]: Further feedback on Tony's feedback:

- It's "Flutter web" NOT "Flutter Web"
- "To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii].
We intend for search engines to find and index this site."  =>
  To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii],
 hoping that search engines find and index this site."

**Review by sfshaza2 — APPROVED** [review]
Minor tweaks, Yegor, but I LOVE this clever idea! I'll go ahead and approve and you can land this whenever you are ready.

**Comment by johnpryan** [comment]
Let's hold off on merging this until we can find a solution for b/339684234

File: src/content/platform-integration/web/faq.md
Comment (yjbanov) [reply src/content/platform-integration/web/faq.md]: Done. Thank you!

**Comment by sfshaza2** [comment]
Can we get this landed, @yjbanov?

**Comment by kevmoo** [comment]
I think we're okay to land now, @sfshaza2 !

**Pull request merged**
